### DISCUSSION TITLE
fine-tuned GPT4o mini model

### DISCUSSION THREAD

**Discussion by liniiiiii** [VUB-HYDR/Wikimpacts#58] [Announcements]
fine-tuned GPT4o mini model: 
version: 20210801
```shell

ft:gpt-4o-mini-2024-07-18:vrije-universiteit-brussel::9rMSmEeA
```

**Comment by i-be-snek** [comment]
Could you add more details? What does this mean for us?

**Reply by liniiiiii** [reply]
Hi, I mark the version of this model, more details will be added by Murathun in the future if needed
